db/seeds.rb

```diff
@@ -61,15 +61,16 @@
     fname = Faker::Name.unique.first_name
     # Including "\\:/" to help with identifying local issues with
     # character escaping.
-    lname = Faker::Name.unique.last_name + "\\:/"
-    name = [fname, "\"The #{fname}\"", lname].join(" ")
+    lname = Faker::Name.unique.last_name
```

Comment:
interestingly, this "unique" call makes testing this harder (you can't call `1000.times { Faker::Name.unique.last_name }`)

About 2% of last names are longer than 10 characters, and very few first names

```
 10000.times { name = Faker::Name.first_name; names.push(name) if name.length > 10 }
=> 10000
 names.uniq
=> ["Charlesetta", "Providencia", "Christopher", "Alejandrina", "Guillermina", "Jackqueline"]
```
Anyway, if you sometimes added 10, 10, and 10-12 you'd hit 30, makes sense.